scripts/src/main/resources/scripts/devon.properties

```diff
@@ -4,7 +4,7 @@
 # For details see https://github.com/devonfw/ide/blob/master/documentation/configuration.asciidoc
 #************************************************************************************************
 
-DEVON_IDE_TOOLS=(java mvn eclipse vscode node npm ng)
+DEVON_IDE_TOOLS=(java mvn eclipse vscode node npm ng ionic)
```

Comment:
we are extending the defaults more and more while we get the feedback that we should rather follow opt-in than opt-out approach.